tests/components/tuya/snapshots/test_climate.ambr

```diff
@@ -50,7 +50,7 @@
   StateSnapshot({
     'attributes': ReadOnlyDict({
       'current_temperature': 22.0,
-      'fan_mode': 1,
+      'fan_mode': None,
```

Comment:
Fan mode is now properly validated.
1 is not a string, so not part of the enum, and now returns `None`.